src/extension.ts

```diff
@@ -391,7 +436,12 @@ function logNotification(message: string, ...items: string[]) {
 function onConfigurationChange() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this function to reduce its Cognitive Complexity from 16 to the 15 allowed. ([typescript:S3776](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3776))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=18&issues=AWMrwwuPau6S9RpGg6m1&open=AWMrwwuPau6S9RpGg6m1)

[ISSUE_KEY]: AWMrwwuPau6S9RpGg6m1